resolver/resolver.h

```diff
@@ -26,6 +26,7 @@ class Resolver final {
     static void finalizeAncestors(core::GlobalState &gs);
     static void finalizeSymbols(core::GlobalState &gs);
     static void computeLinearization(core::GlobalState &gs);
+    static void computeExternalTypes(core::GlobalState &gs);
```

Comment:
Does this need to be exposed, or can it live in an anonymous namespace in resolver.cc?

Reply:
Why would it need to be exposed? It's only called within `resolver.cc`. I added it here as that's where `finalizeAncestors` and friends are defined.

I actually don't know why some of the private passes are exposed in `resolver.h`.

Reply:
Well, maybe we can address that in a follow-up at some point :)

Reply:
Oh okay wait I realize what you're saying. Yeah I can make it anonymous.

Reply:
I think they are in `resolver.h` bc they are defined in separate files.